etc/templates/config/generic/syscheck.manager.template

```diff
@@ -11,7 +11,7 @@
     <alert_new_files>yes</alert_new_files>
 
     <!-- Don't ignore files that change more than 3 times -->
-    <auto_ignore>no</auto_ignore>
+    <auto_ignore frecuency="10" timeframe="60">no</auto_ignore>
```

Comment:
A frequency of 10 changes per hour is very high. Let's set 10 changes per hour by default.